nistats/tests/test_dmtx.py

```diff
@@ -159,59 +157,29 @@ def test_design_matrix0d():
     assert_equal(X.shape[1], 8)
 
 
-def test_design_matrix1():
-    # basic test based on basic_paradigm and glover hrf
-    tr = 1.0
-    frame_times = np.linspace(0, 127 * tr, 128)
-    paradigm = basic_paradigm()
-    hrf_model = 'glover'
-    X, names = design_matrix_light(frame_times, paradigm, hrf_model=hrf_model,
-                            drift_model='polynomial', drift_order=3)
-    assert_equal(len(names), 7)
-
```

Comment:
all the matrix1* tests are now together in test_design_matrix1 in line 172. There were basically 4 tests duplicating all lines of code except the last one. I did not see the point of having 4 separate tests.